src/script.c

```diff
@@ -541,34 +541,46 @@ int wally_scriptpubkey_p2sh_from_bytes(
     return ret;
 }
 
+int pubkey_compare(const void *a, const void *b)
+{
+    return memcmp(a, b, EC_PUBLIC_KEY_LEN);
+}
+
 int wally_scriptpubkey_multisig_from_bytes(
     const unsigned char *bytes, size_t bytes_len, uint32_t threshold,
     uint32_t flags, unsigned char *bytes_out, size_t len, size_t *written)
 {
     size_t n_pubkeys = bytes_len / EC_PUBLIC_KEY_LEN;
     size_t script_len = 3 + (n_pubkeys * (EC_PUBLIC_KEY_LEN + 1));
     size_t i;
+    unsigned char pubkey_bytes[bytes_len];
```

Comment:
use of VLA is causing build failures and we want to avoid them in the codebase.

Reply:
I changed it to the maximum size needed for 16 pubkeys.